service/matching/taskListManager.go

```diff
@@ -372,13 +378,24 @@ func (c *taskListManagerImpl) GetTaskContext(
 		WorkflowId: common.StringPtr(task.WorkflowID),
 		RunId:      common.StringPtr(task.RunID),
 	}
+
+	// reload from domainCache in case it got empty result during construction
+	if len(c.domainName) == 0 {
```

Comment:
c.domainName != "" is the go standard way

Reply:
I don't like arguing about this. But still wondering where it comes from as standard. 
https://stackoverflow.com/questions/18594330/what-is-the-best-way-to-test-for-an-empty-string-in-go

And we do have both in our codebase.
So I will change it anyway if you think it is needed.

Reply:
this isn't a blocker and yes, we have both in our code base today, which is unfortunate